ontology/yaml/resources/HVAC/entity_types/TK.yaml

```diff
@@ -0,0 +1,69 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+########################
+### Canonical Types  ###
+########################
+
+### Tanks ###
+
+TK_SWTC_PHWTC:
+  description: "Tank with water temperature monitoring"
+  is_canonical: true
+  implements:
+  - TK  
+  - SWTC
+  - PHWTC
+  
+TK_TLM:  
+  description: "Tank with water level monitoring"
+  is_canonical: true
+  implements:
+  - TK
+  - TLM 
+
+TK_SPR_PLPM:  
+  description: "Tank with pipeline pressure monitoring for sprinkler pipeline"
+  is_canonical: true
+  implements:
+  - TK
+  - PLPM
+
+TK_HDR_PLPM:  
+  description: "Tank with pipeline pressure monitoring for hydrant tank pipeline"
+  is_canonical: true
+  implements:
+  - TK
+  - PLPM  
+
+TK_FHT_TLM:  
+  description: "Tank with level monitoring for fire hydrant tanks"
```

Comment:
same as above, not important if it serves fire hydrants. however, it should be in the SAFETY namespace.

Reply:
Moved to FACP pull request #634

Reply:
Created the canonical under safety TK_LM for fire hydrant tank level monitoring.